.github/workflows/deploy-dev.yml

```diff
@@ -120,29 +120,28 @@ jobs:
             --set image.tag=$IMAGE_TAG
             --set ckan.psql.initialize=false
             --set solr.initialize.enabled=false
-  notifyfail:
-    runs-on: ubuntu-latest
-    if: ${{ failure() }}
+
+  integrationtest:
+    if: ${{ success() }}
     needs:
       - ui
       - ckan
+    runs-on: ubuntu-latest
     steps:
-      - name: Slack Notification
-        uses: rtCamp/action-slack-notify@v2
-        env:
-          SLACK_USERNAME: github-bot
-          SLACK_CHANNEL: developers
-          SLACK_COLOR: danger
-          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
-          SLACK_TITLE: "dev deployment: failed"
-          SLACK_MESSAGE: "UI: ${{needs.ui.result}}\nCKAN: ${{needs.ckan.result}}"
-          MSG_MINIMAL: "true"
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 16.x
+          cache: 'npm'
+          cache-dependency-path: ui/package-lock.json
```

Comment:
Ooo! 💘

Reply:
It seemed more sensible than having a one entry matrix.